benchmarks/6.0.0/6.0.0_single-node_2-core_jdk-11.md

```diff
@@ -0,0 +1,180 @@
+| Implicit Grant Redirect With Consent | Obtain an access token using the OAuth 2.0 implicit grant type. |
+| Password Grant Type | Obtain an access token using the OAuth 2.0 password grant type. |
+| Client Credentials Grant Type | Obtain an access token using the OAuth 2.0 client credential grant type. |
+| OIDC Auth Code Grant Redirect With Consent | Obtain an access token and an id token using the OAuth 2.0 authorization code grant type. |
+| OIDC Implicit Grant Redirect With Consent | Obtain an access token and an id token using the OAuth 2.0 implicit grant type. |
+| OIDC Password Grant Type | Obtain an access token and an id token using the OAuth 2.0 password grant type. |
+| OIDC Auth Code Request Path Authenticator With Consent | Obtain an access token and an id token using the request path authenticator. |
+| SAML2 SSO Redirect Binding | Obtain a SAML 2 assertion response using redirect binding. |
+
+Our test client is [Apache JMeter](https://jmeter.apache.org/index.html). We test each scenario for a fixed duration of
+time and split the test results into warm-up and measurement parts and use the measurement part to compute the
+performance metrics. For this particular instance, the duration of each test is **10 minutes** and the warm-up period is **2 minutes**.
+
+We run the performance tests under different numbers of concurrent users and heap sizes to gain a better understanding on how the server reacts to different loads.
+
+The main performance metrics:
+
+1. **Throughput**: The number of requests that the WSO2 Identity Server processes during a specific time interval (e.g. per second).
+2. **Response Time**: The end-to-end latency for a given operation of the WSO2 Identity Server. The complete distribution of response times was recorded.
+
+In addition to the above metrics, we measure the load average and several memory-related metrics.
+
+The following are the test parameters.
+
+| Test Parameter | Description | Values |
+| --- | --- | --- |
+| Scenario Name | The name of the test scenario. | Refer to the above table. |
+| Heap Size | The amount of memory allocated to the application | 2G |
+| Concurrent Users | The number of users accessing the application at the same time. | 50, 100, 150, 300, 500 |
+| IS Instance Type | The AWS instance type used to run the Identity Server. | [**c5.large**](https://aws.amazon.com/ec2/instance-types/) |
+| IS Instance Type | The AWS RDS instance type used to run the Identity Server. | [**db.m4.2xlarge**](https://aws.amazon.com/rds/instance-types/) |
+| JDK version | The JDK version used to run the Identity Server. | JDK 11.0.15.1  |
+
+In order to ease cross-comparison with earlier versions of the product, the following update has been made to the  deployment.toml.
+
+[authentication_policy]
+
+disable_account_lock_handler=true
+
+[identity_mgt.events.schemes.WorkflowPendingUserAuthnHandler]
+
+subscriptions=[]
+
+[authorization_manager.properties]
+
+GroupAndRoleSeparationEnabled=false
```

Comment:
let's move this to code block, so it will have a better visualization

Reply:
This is resolved with https://github.com/wso2/performance-is/pull/122/commits/c2a7eaac06de753d81e234e1855789ffd047ed09